vpr/src/base/vpr_types.h

```diff
@@ -113,6 +113,7 @@ enum class e_router_lookahead {
     CLASSIC,      ///<VPR's classic lookahead (assumes uniform wire types)
     MAP,          ///<Lookahead considering different wire types (see Oleg Petelin's MASc Thesis)
     EXTENDED_MAP, ///<Lookahead with a more extensive node sampling method
+    CONNECTION_BOX_MAP, ///<Lookahead considering different wire types and IPIN connection box.
```

Comment:
We should put this after NO_OP so that NO_OP's enum value is the same in upstream and downstream VTR.

Reply:
Done